proxy.go

```diff
@@ -188,6 +216,207 @@ func (proxy *proxy) releaseToken(token Token) (*tokenInfo, error) {
 	return info, nil
 }
 
+var storeStateDir = "/var/lib/clear-containers/proxy/"
+
+type proxyStateOnDisk struct {
```

Comment:
Again, I'd prefer to see all new types at the top of the file (or even in a separate `types.go`).